### PR TITLE
Update libraries

### PR DESCRIPTION
**Pull request by kordianbruck**
By specifying dependencies individually we can update them using our bot

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -79,9 +79,6 @@ android {
 
 def androidSupportVersion = '27.1.1'
 def retrofitVersion = '2.3.0'
```
Comment (pfent) [inline app/build.gradle]: what about those?

File: app/build.gradle [hunk 2]
```diff
@@ -79,9 +79,6 @@ android {
...
-def firebaseVersion = '15.0.0'
```
Comment (thellmund) [inline app/build.gradle]: Why did you remove those?

File: app/build.gradle (hunk 1 above)
Comment (thellmund) [reply app/build.gradle]: GSON is also available in a new version and can be updated.

Comment (kordianbruck) [reply app/build.gradle]: Dependabot should handle those.

Comment (pfent) [reply app/build.gradle]: so why do we need to specify the other dependencies individually, but those are OK being specified as variables?

Comment (kordianbruck) [reply app/build.gradle]: Because only firebase and android architecture components announced, that they will no longer simultaneously update their libs.
